aiohttp_security/jwt_identity.py

```diff
@@ -19,6 +19,17 @@
 AUTH_SCHEME = 'Bearer '
 
 
+if HAS_JWT:
+    # This class inherits from ValueError to maintain backward compatibility
+    # with previous versions of aiohttp-security.
+    class JWTIdentityError(jwt.exceptions.PyJWTError, ValueError):
```

Comment:
I think we could do `_jwt_bases  = (jwt.exceptions.PyJWTError, ValueError)` in the above try/except, and then just define this unconditionally (`class JWTIdentityError(*_jwt_bases):`).